### PR TITLE
Topic/update dependency

### PR DESCRIPTION
**Pull request by gosunuts**
change protobuf file name ( raft.proto -> aergo_raft.proto )
need to regenerate proto about dependency repository 

- [x] https://github.com/aergoio/aergo
- [x] https://github.com/aergoio/aergo-indexer-2.0
- [ ] https://github.com/aergoio/heraj
- [ ] https://github.com/aergoio/herapy
- [ ] https://github.com/aergoio/herajs
- [ ] https://github.com/aergoio/heraphp

### CODE REVIEW

File: proto/account.proto [hunk 1]
```diff
@@ -1,7 +1,7 @@
 syntax = "proto3";
 
 package types;
-//option go_package = "github.com/aergoio/aergo/types";
+option go_package = "/types";
```
Comment (kroggen) [inline proto/account.proto:4]: Why this change? Was it tested with this change?

Comment (gosunuts) [reply proto/account.proto:4]: previous code only worked only for Aergo repositories. ( In fact, it didn't work even in aergo )
[aergo repository](https://github.com/aergoio/aergo/tree/topic/update-go-dependency) also apply this change too. ( make protoc / make protoclean )

Comment (gosunuts) [reply proto/account.proto:4]: it is all tested include sync test
https://github.com/aergoio/aergo/blob/0ce27749e9d09998aedc5bc35654961711dc89a0/CMakeLists.txt#L56

**Review by kroggen — APPROVED** [review]
